src/OrbitGl/App.cpp

```diff
@@ -2417,7 +2417,7 @@ const InstrumentedFunction* OrbitApp::GetInstrumentedFunction(uint64_t function_
   return HasCaptureData() ? GetCaptureData().GetInstrumentedFunctionById(function_id) : nullptr;
 }
 
-void OrbitApp::SetVisibleFunctionIds(absl::flat_hash_set<uint64_t> visible_function_ids) {
+void OrbitApp::SetVisibleScopeIds(absl::flat_hash_set<uint64_t> visible_function_ids) {
   data_manager_->set_visible_function_ids(std::move(visible_function_ids));
```

Comment:
This is renamed but the parameter is still called `visible_function_ids` and the implementation still calls `DataManager::set_visible_function_ids`. Which is it?

Reply:
Good point. These are all scope ids.